build/grunt.js

```diff
@@ -405,6 +410,14 @@ module.exports = function(grunt) {
         'watch:babel',
         'browserify:tests'
       ],
+      dev: [
```

Comment:
we specifically don't run `watch:minify` because it is a time consuming process.